geomstats/information_geometry/multivariate_normal.py

```diff
@@ -0,0 +1,348 @@
+"""Statistical Manifold of multivaraite normal distributions with the Fisher metric.
+
+Lead author: Antoine Collas.
+"""
+
+import math
+
+from scipy.stats import multivariate_normal, norm
+
+import geomstats
+import geomstats.backend as gs
+from geomstats.geometry.base import OpenSet
+from geomstats.geometry.euclidean import Euclidean
+from geomstats.geometry.riemannian_metric import RiemannianMetric
+from geomstats.information_geometry.base import InformationManifoldMixin
+from geomstats.information_geometry.normal import NormalMetric
+
+
+class MultivariateDiagonalNormalDistributions(OpenSet, InformationManifoldMixin):
+    """Class for the manifold of diagonal multivariate normal distributions."""
```

Comment:
Add some information in the docstring, in particular: what is the shape of a point in this class ? If it is a vector of size $2n$, what does it contain, and in what order ?

e.g.
```
This is the class for multivariate normal distributions on the $n$-dimensional Euclidean
space with diagonal covariance matrices. Each distribution is represented by a vector
of size $2n$ where the first $n$ elements contain the mean vector and the $n$ last
elements contain the diagonal of the covariance matrix.

Parameters
----------
n : int
      Integer representing the number of variables of the multivariate normal distributions.
```